services/src/unit-test/java/uk/gov/moj/sdt/services/messaging/IndividualRequestMdbTest.java

```diff
@@ -153,11 +140,20 @@ public void readMessageDataAccessException() throws JMSException {
     public void readMessageInvalidObject() throws JMSException {
         individualRequestMdb = new IndividualRequestMdb(mockTargetSubmissionService);
 
-        final Message message = EasyMock.createMock(Message.class);
+        final Message message = mock(Message.class);
 
         individualRequestMdb.readMessage(message);
 
-        Assert.assertTrue("Test successfully completed", true);
+        assertTrue(true,TEST_SUCCESSFULLY_COMPLETED);
+    }
+
+    @Test
+    void setMockTargetSubmissionServiceTest(){
+        IndividualRequestMdb individualRequestMdbMock = mock(IndividualRequestMdb.class);
+
+        individualRequestMdbMock.setTargetAppSubmissionService(mockTargetSubmissionService);
+
```

Comment:
Add an assertEquals() here to check that the target submission service has been set to the expected object.

Reply:
how would we do this if there is no get method for the TargetAppSubmissionService?

Reply:
It's a bit awkward, but you can use the getAccessibleField() method on the AbstractSdtUnitTestBase class to get the private field.  See setMockSdtBulkReferenceGeneratorTest() in BulkSubmissionServiceTest for an example of this.

Reply:
done